configure

```diff
@@ -156,14 +156,15 @@ my @specs = # Array of array-refs, each inner array is an option group which
     'wireshark_build:s', 'Wireshark CMake Build Location',
     'wireshark_lib:s', 'Wireshark CMake libraries location, relative to wireshark_build',
     'glib:s', 'GLib for Wireshark (use GLIB_ROOT or system pkg)',
-    'rapidjson', 'RapidJSON for Wireshark Sample Dissection(optional)',
+    'rapidjson:s', 'RapidJSON for Wireshark Sample Dissection(optional)',
```

Comment:
I realize it was there before, but I don't see why the space is missing before (optional).